### ISSUE TITLE
can't  finish the quest.

### ISSUE THREAD

**Issue by lotu5david** [phoomin2012/travian-kingdom#16]
![Image](http://2stars.cc/10.jpg)

**Comment by lotu5david** [comment]
the image button is not available.

**Comment by lotu5david** [comment]
oh it works . but it was so slowly...

**Comment by phoomin2012** [comment]
Because of cache system send wrong data. In process to fix it

**Comment by lotu5david** [comment]
yes  when i finished a quest. the website should be freshed automatically. but they didn't. i mush refresh the website manually

**Comment by lotu5david** [comment]
does it easy to fix?

**Comment by lotu5david** [comment]
Also the php.exe used about 95% of cpu. Does it necessary?
